R/tags.R

```diff
@@ -843,11 +843,16 @@ names(known_tags) <- known_tags
 #' @keywords NULL
 #' @import rlang
 tags <- lapply(known_tags, function(tagname) {
-  function(..., .noWS=NULL) {
-    validateNoWS(.noWS)
-    contents <- dots_list(...)
-    tag(tagname, contents, .noWS=.noWS)
-  }
+  # Overwrite the body with the `tagname` value injected into the body
+  new_function(
+    args = exprs(... = , .noWS = NULL),
+    expr({
+      validateNoWS(.noWS)
+      contents <- dots_list(...)
+      tag(!!tagname, contents, .noWS=.noWS)
+    }),
+    env = asNamespace("htmltools")
```

Comment:
Does the namespace exist when this is run? Note that this may differ between `load_all()` and a real package building scenario.

Reply:
Note: I just tested and `asNamespace("htmltools")` does seem to work during package build.